packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -44,6 +48,23 @@ const AppContents = ({
         }
     }, [is_logged_in, is_window_loaded]);
 
+    const onCancelDialog = () => {
+        setDialogStatus(false);
+    };
+    const onConfirmDialog = () => {
+        setDialogStatus(false);
+        window.open(external_link);
+    };
+    React.useEffect(() => {
+        document.addEventListener('click', function(e) {
+            if (e.target.relList && e.target.relList.value !== '') {
```

Comment:
The check for external link is missing here. We are applying this for all links, not just the external ones.